api/src/Entity/User.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\UserRepository")
+ */
+class User implements UserInterface
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id()
```

Comment:
Not using empty parentesis is better for DocParser performances cause it'll prevent any useless analysis of arguments:
https://github.com/doctrine/annotations/blob/580f6eb30e375805c7dd8f739efd3b95a39a79f8/lib/Doctrine/Annotations/DocParser.php#L836-L842

Reply:
Now that's really nitpicking, but okay 😛

Reply:
BTW, id should be uuid, according the other entities in this project:
https://github.com/api-platform/demo/blob/ea79710c4342529aa175d4bc93cb2a719f8ed0c6/api/src/Entity/Book.php#L34-L42